tasks/Ultimatum game (various versions).md

```diff
@@ -0,0 +1,41 @@
+# Summary
+>  One player, the proposer, is endowed with a sum of money and can split it with another player (the responder). The responder then can accept or reject the money. 
+
+# References
+> Task Description: https://en.wikipedia.org/wiki/Ultimatum_game
+
+# Stimuli
+## The visual components
+> Document or generate the visual stimuli seen by participants when performing this task. Provide screenshots or illustrations.
+
+## Materials for alternative versions of the experiment 
+>  competitive ultimatum game : there are many proposers and the responder can accept at most one of their offers: With more than three (naïve) proposers the responder is usually offered almost the entire endowment 
+  
+ > ultimatum game with tipping: a tip is allowed from responder back to proposer, a feature of the trust game, and net splits tend to be more equitable.
+   
+ > reverse ultimatum game: gives more power to the responder by giving the proposer the right to offer as many divisions of the endowment as they like. Now the game only ends when the responder accepts an offer or abandons the game, and therefore the proposer tends to receive slightly less than half of the initial endowment
+   
+ > Incomplete information ultimatum games: Some authors have studied variants of the ultimatum game in which either the proposer or the responder has private information about the size of the pie to be divided
+   
+ > The pirate game illustrates a variant with more than two participants with voting power
+   
```

Comment:
I found some instructions for a basic ultimatum game if you want to include them:

Rounds and Matchings: The experiment consists of a number of rounds. Note: You will be matched with the same person in all rounds.

Decisions: One of you will be designated as a Proposer (initial decision-maker) who will begin the round by proposing a division of an amount of money, $10.00. The other person, designated as responder, is told of the proposed division and must either accept it or not.

Role: You have been randomly assigned to be a Proposer (or initial decision-maker) in this process, and you will begin by suggesting amounts of money for each of you that sum to $10.00.

OR Role: You have been randomly assigned to be a Responder in this process. The other person (proposer) will begin by suggesting amounts of money for each of you that sum to $10.00.

Earnings: If the responder accepts the proposed division, then each person earns their part of the proposed division. If the responder rejects, both earn $0.00. 

Please remember that you will be matched with the same person in all rounds.

The proposer (initial decision-maker) in each pair suggests a division of the $10.00; this proposal determines earnings for the round if it is accepted. A rejection produces earnings of $0.00 for each person. If the proposal is rejected, earnings are $0.00 for both proposer and responder.

There will be a number of rounds, and you are always matched with the same person.